src/plugins/bulk_update.py

```diff
@@ -0,0 +1,65 @@
+import flux
```

Comment:
Should this file have a copyright header? I'm actually unsure, but might as well throw it in here just in case.

Reply:
Oh yeah, thank you for pointing this out. I just pushed up a fix that adds a copyright header. Sorry for forgetting to put one. 🤦‍♂️